Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -480,16 +544,38 @@ - (void)finishWithContext:(BPExecutionContext *)context {
             context.finalExitStatus = BPExitStatusAppCrashed;
             NEXT([self proceed]);
             return;
+        case BPExitStatusSimulatorDeleted:
+            self.finalExitStatus = BPExitStatusSimulatorDeleted;
+            self.exitLoop = YES;
+            return;
     }
 
 }
 
+- (void)writeDeviceIDFile {
```

Comment:
Agreed. I don't think something more than a temp file is warranted yet.
Though this does have me wondering if `bluepill` should fork() to execute `bp` as a child process instead... it would make that sort of communication in the future easier, and might also solve the command line arg processing differences (or maybe it complicates it).